src/rs232.rs

```diff
@@ -110,7 +110,9 @@ impl RS232 {
 
         for _ in 1..3 {
             if let Ok(_) = self.read_line(&mut buffer) {
-                if buffer.starts_with("$GP") {
+                if buffer.len() >= 15 && buffer.starts_with("$G") &&
+                    buffer.chars().nth(6).unwrap() == ','
```

Comment:
rustfmt puts it like this? which version is this?

Reply:
Stable, which self-identifies as deprecated but there is no newer stable release. It suggested I use nightly releases, but I assumed you had not reformat the entire project with the suggestions from an unstable style tool.

Reply:
ah ok, good thinking. I've been using the latest stable that's not deprecated: 0.9.0. I'll suggest to use that instead of 0.10.0. `cargo install -f --vers 0.9.0 rustfmt` should do the trick I think.

Reply:
I get the exact same formatting with 0.9.0 as with 0.10.0. Have you got any local configuration that affects this? What formatting do you expect here?

Reply:
ah ok, no i just wanted to be sure it's correct. I didn't try on my end. Nevermind. I'll merge now. :)